pumps/elasticsearch.go

```diff
@@ -355,12 +370,50 @@ func (e *ElasticsearchPump) Init(config interface{}) error {
 		e.log.Info("Index will have date appended to it in the format ", e.esConf.IndexName, "-YYYY.MM.DD")
 	}
 
+	if e.esConf.UseSSL {
+		err := e.setTLSConfig()
+		if err != nil {
+			e.log.Fatal("Failed to set TLS config: ", err)
+		}
+	}
+
 	e.connect()
 
 	e.log.Info(e.GetName() + " Initialized")
 	return nil
 }
 
+// setTLSConfig sets the TLS config for the pump
+func (e *ElasticsearchPump) setTLSConfig() error {
```

Comment:
@mativm02 Wouldn't be easier to create a helper function that receives SSLCertFile, SSLKeyFile, and SSLInsecureSkipVerify and returns a `&tls.Config`? IMO it's easier to test.

Reply:
done!